modules/ROOT/pages/customize-nic.adoc

```diff
@@ -0,0 +1,49 @@
+:experimental:
+= How to Customize a NIC Name
+
+== Using a systemd Link File
+You can create a systemd xref:https://www.freedesktop.org/software/systemd/man/systemd.link.html[link file] with Ignition configs.
+
+For example, to name NIC with the MAC address `12:34:56:78:9a:bc` to "infra", place a systemd link file at `/etc/systemd/network/25-infra.link` using the xref:fcct-config.adoc[FCCT] configuration snippet shown below:
+
+.Example: Customize NIC via systemd Link File
+[source,yaml]
+----
+storage:
+  files:
+    - path: /etc/systemd/network/25-infra.link
+      mode: 0644
+      contents:
+        inline: |
+          [Match]
+          MACAddress=12:34:56:78:9a:bc
+          [Link]
+          Name=infra
+----
+
+== Using Udev Rules
+Similarly, also through Ignition configs, to name NIC with the MAC address `12:34:56:78:9a:bc` to "infra", create a xref:https://linux.die.net/man/8/udev[udev rule] at `/etc/udev/rules.d/80-ifname.rules` using the xref:fcct-config.adoc[FCCT] configuration snippet shown below:
+
+.Example: Customize NIC via Udev Rules
+[source,yaml]
+----
+storage:
+  files:
+    - path: /etc/udev/rules.d/80-ifname.rules
+      mode: 0644
+      contents:
+        inline: |
+          SUBSYSTEM=="net", ACTION=="add", DRIVERS=="?*", ATTR{address}=="12:34:56:78:9a:bc", ATTR{type}=="1", NAME="infra"'
+----
+
+== Networking in the Initramfs via Kernel Arguments
+If networking in the initramfs is required, the Dracut argument `ifname=` dynamically creates a udev rule to change the name of a NIC.
```

Comment:
It feels like we might need to give the user some more background information. Maybe something like:

```suggestion
If networking in the initramfs is required, the kernel argument `ifname=` will dynamically create a udev rule to change the name of a NIC in the initramfs. If the custom name needs to be applied to the real root then a systemd .link file or udev rule, as discussed earlier on this page, will need to get created via Ignition. See [this issue](https://github.com/coreos/fedora-coreos-tracker/issues/553) for more details.
```

Reply:
ahh. I see this information is in the last paragraph. Maye we can add some of this down there and add a link to the issue down there so people can get more context on why they need to do both.